site/specs/numbers.json

```diff
@@ -12549,69 +12595,115 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>updates the contents of a site id</p>\n",
-        "operationId": "PUT /accounts/{accountId}/sites/{siteId}",
+        "description": "Updates the contents of site",
+        "summary": "Update Site",
+        "operationId": "UpdateSite",
         "parameters": [
           {
             "name": "accountId",
-            "description": "",
+            "description": "User's account ID",
             "in": "path",
             "required": true,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
-            "example": null
+            "example": "1234567"
           },
           {
             "name": "siteId",
-            "description": "",
+            "description": "Site's ID for which information will be updated",
             "in": "path",
             "required": true,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
-            "example": null
+            "example": "407"
           }
         ],
         "requestBody": {
           "content": {
             "application/xml": {
+              "schema": {
+                "title": "CreateSite",
+                "properties": {
+                  "Name": {
+                    "description": "The name of the site. Max length restricted to 10 characters",
+                    "type": "string"
```

Comment:
Should we use `maxLength` to restrict it to 10 characters?

https://swagger.io/docs/specification/data-models/data-types/#string